lib/OpenQA/Constants.pm

```diff
@@ -0,0 +1,73 @@
+package OpenQA::Constants;
+use strict;
+
+# Minimal worker version that allows them to connect;
+# To be modified manuallly when we want to break compability and force workers to update
+# If this value differs from server to worker then it won't be able to connect.
+use constant WEBSOCKET_API_VERSION => 1;
+
+# How many jobs to allocate in one tick. Defaults to 50 ( set it to 0 for as much as possible)
+use constant MAX_JOB_ALLOCATION => $ENV{OPENQA_SCHEDULER_MAX_JOB_ALLOCATION} // 50;
```

Comment:
-1 for me, scheduler constants should stay there, since they are not meant to be required by other packages outside of it. We allow to override them by env variable - because you *might* want to set them while running the service.

Reply:
So from those the variables only 

> WEBSOCKET_API_VERSION
> WORKERS_CHECKER_THRESHOLD
> STATUS_UPDATES_SLOW
> STATUS_UPDATES_FAST

?

Reply:
yeah, i would just like to avoid we start use this file for all the constants we have, but only those that are meant to be exported to other packages outside of their namespace or have a shared meaning.

I think you can also extract  [MAX_TIMER / MIN_TIMER ]( https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Worker/Common.pm#L62) too since are tied to WORKERS_CHECKER_THRESHOLD .